beitragsordnung.md

```diff
@@ -2,13 +2,16 @@
 
 Die Mitgliederversammlung des Vereins Welcome Werkstatt e.V. hat am 9.11.2019 folgende Beitragsordnung beschlossen:
 
-1. Alle Vereinsmitglieder – sowohl Fördermitglieder als auch ordentliche Mitglieder mit Stimmrecht – zahlen einen Mitgliedsbeitrag. Der Mitgliedsbeitrag wird monatlich berechnet.
+1. Miedergebühren
+1.1. Alle Vereinsmitglieder – sowohl Fördermitglieder als auch ordentliche Mitglieder mit Stimmrecht – zahlen einen Mitgliedsbeitrag. Der Mitgliedsbeitrag wird monatlich berechnet.
 
-2. Ehrenmitglieder sind von der Beitragszahlung befreit. In Sonderfällen können gemäß Satzung § 6 auch von ordentlichen Mitgliedern mit Stimmrecht begründete Anträge auf Ermäßigung der Beitragshöhe gestellt werden.
+1.2. Ehrenmitglieder sind von der Beitragszahlung befreit. 
 
-3. Die Beitragszahlung der Mitglieder erfolgt entweder per Überweisung auf das Vereinskonto oder im Lastschriftverfahren. Sie erfolgt, je nach gewählter Zahlungsweise des Mitglieds, jeweils zum ersten Werktag des Monats oder im ersten Monat des laufenden Quartals, des Halbjahres oder des Jahres. Wird das Lastschriftverfahren gewählt, so erteilt das Mitglied dem Verein hierfür ein SEPA-Lastschriftmandat.
+1.3. In Sonderfällen können gemäß Satzung § 6 auch von ordentlichen Mitgliedern mit Stimmrecht begründete Anträge auf Ermäßigung der Beitragshöhe gestellt werden.
```

Comment:
In § 6 der Satzung steht nichts von der Möglichkeit Anträge zu stellen, sondern nur, dass der Vorstand eine Einzelfallregelung per Mehrheitsbeschluss festlegen kann. 
Ich versteh leider nicht, was genau 1.3 regeln soll - dass man Anträge auf Ermäßigung stellen kann oder dass auch ordentliche Mitglieder ermäßigte Beiträge zahlen könnten?

Reply:
Jedes Mitglied könnte einen Antrag auf Ermäßigung stellen. Gemeint ist letztlich eine Ermäßigung des Beitrags falls ein Mitglied Arbeitslos wird oder ander Gründe vorliegen.

Reply:
Dann kann der Punkt aus der Beitragsordnung raus, da er ja in der Satzung geregelt wird.

Reply:
Ich habe die Regelung nochmal etwas besser formuliert und würde den eigentlich gerne als Präzisierung drin lassen.